sections/contact-form.liquid

```diff
@@ -17,13 +17,20 @@
 <div class="color-{{ section.settings.color_scheme }} gradient">
   <div class="contact page-width page-width--narrow section-{{ section.id }}-padding">
     {%- if section.settings.heading != blank -%}
-      <h2 class="title title-wrapper--no-top-margin inline-richtext {{ section.settings.heading_size }}">
+      <h2 class="title title-wrapper--no-top-margin inline-richtext {{ section.settings.heading_size }}{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
         {{ section.settings.heading }}
       </h2>
     {%- else -%}
       <h2 class="visually-hidden">{{ 'templates.contact.form.title' | t }}</h2>
     {%- endif -%}
-    {%- form 'contact', id: 'ContactForm', class: 'isolate' -%}
+    {%- liquid
+      if settings.animations_reveal_on_scroll
+        assign contact_form_scroll_trigger_class = 'isolate scroll-trigger animate--slide-in'
+      else
+        assign contact_form_scroll_trigger_class = 'isolate'
+      endif
+    -%}
```

Comment:
Here I'd say that we can call the variable something more general. Since it's a scroll trigger class only when the setting is ON. 
So we could just do `contact_form_classes`

```suggestion
    {%- liquid
      assign contact_form_class = 'isolate'
      if settings.animations_reveal_on_scroll
        assign contact_form_class = 'isolate scroll-trigger animate--slide-in'
      endif
    -%}
```